pom.xml

```diff
@@ -150,10 +148,36 @@
 				<version>${junit.version}</version>
 				<scope>test</scope>
 			</dependency>
-
 		</dependencies>
 	</dependencyManagement>
 
+	<profiles>
+		<profile>
+			<id>default</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<properties>
+				<junit.includedTags/>
+				<junit.excludedTags>WebUI,Integration</junit.excludedTags>
+			</properties>
+		</profile>
+		<profile>
+			<id>webui</id>
+			<properties>
+				<junit.includedTags>WebUI</junit.includedTags>
+				<junit.excludedTags>Integration</junit.excludedTags>
+			</properties>
+		</profile>
+		<profile>
+			<id>integration</id>
+			<properties>
+				<junit.includedTags>Integration</junit.includedTags>
+				<junit.excludedTags>WebUI</junit.excludedTags>
+			</properties>
+		</profile>
+	</profiles>
```

Comment:
I introduced profiles to make it more clear and easy to use and select which profile you'd like to use. By default, the first profile is used.